tests/test_snippets/test_level_commands.py

```diff
@@ -41,9 +38,9 @@ def collect_snippets(path):
 
 Hedy_snippets = [(s.name, s) for s in collect_snippets(path='../../content/commands')]
 
-lang = 'ar' #useful if you want to test just 1 language
-if lang:
-    Hedy_snippets = [(name, snippet) for (name, snippet) in Hedy_snippets if snippet.language == lang]
+# lang = 'de' #useful if you want to test just 1 language
+# if lang:
+#     Hedy_snippets = [(name, snippet) for (name, snippet) in Hedy_snippets if snippet.language[:2] == lang]
 
```

Comment:
small changes, but commented? This could be deleted.

Reply:
Nah, this is useful for the case when you want to test 1 lang (comment also explains that)